File: ejercicicos con jose cuellar/Python/ejercicio8.py
#Escribe un programa que pida al usuario los siguientes datos: días, horas y minutos. Y le conteste con la cantidad de segundos totales que son esos datos.

day = 86400
hour = 3600
minute = 60

dia = int(input("Ingrese la cantidad de dias: "))
hora = int(input("Ingrese la cantidad de horas: "))
minuto = int(input("Ingrese la cantidad de minutos: "))

segundo = (dia*day) + (hora*hour) + (minuto*minute)

print('La cantidad de segundos totales son',segundo)

File: ejercicicos con jose cuellar/js/js/ejercicio8.js
function calcular(){

    const dias = 86400;
    const horas = 3600;
    const minutos = 60;

    let dia, hora, minuto

    dia = document.getElementById("dia").value 
    hora = document.getElementById("hora").value 
    minuto = document.getElementById("minuto").value 

    segundo = (dia*dias) + (hora*horas) + (minuto*minutos);

    print = document.getElementById("respuesta")
    print.innerHTML = "La cantidad de segundos que corresponden estos datos son " + segundo;
}